jax/export.py

```diff
@@ -12,19 +12,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 __all__ = ["DisabledSafetyCheck", "Exported", "export", "deserialize",
-           "maximum_supported_serialization_version",
-           "minimum_supported_serialization_version",
-           "default_lowering_platform",
+           "maximum_supported_calling_convention_version",
+           "minimum_supported_calling_convention_version",
+           "default_export_platform",
            "SymbolicScope", "is_symbolic_dim",
            "symbolic_shape", "symbolic_args_specs"]
 
 from jax._src.export._export import DisabledSafetyCheck as DisabledSafetyCheck
```

Comment:
Side-note: the `import xxx as xxx` pattern is not necessary if you're using `__all__` to explicitly list all public APIs.

Reply:
I did not know this. Is my change now fine?

Reply:
Yes, looks good!